examples/tutorials/self-paced-training/part-3_security_and_privacy/chapter-6_Security_in_federated_compute_system/06.2_identity_security/identity_security.ipynb

```diff
@@ -0,0 +1,210 @@
+    "First, let's generate a default startup kits using POC mode. POC internally call provision to generate the default startup kits"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": []
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "! echo y | nvflare poc prepare\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "! tree /tmp/nvflare/poc/example_project/prod_00"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Look at the each site's start up kit. We see\n",
+    "\n",
+    "```\n",
+    "startup\n",
+    "│   │   ├── client.crt  <-- certificate\n",
+    "│   │   ├── client.key  <-- private key\n",
+    "│   │   ├── fed_client.json\n",
+    "│   │   ├── rootCA.pem  <-- root certificate\n",
+    "│   │   ├── signature.json\n",
+    "│   │   ├── start.sh\n",
+    "│   │   ├── stop_fl.sh\n",
+    "│   │   └── sub_start.sh\n",
+    "\n",
+    "```\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Authorization\n",
+    "\n",
+    "Federated learning is conducted over computing resources owned by different organizations. Naturally these organizations have concerns about their computing resources being misused or abused. Even if an NVFLARE docker is trusted by participating orgs, researchers can still bring their own custom code to be part of a study (BYOC), which could be a big concern to many organizations. In addition, organizations may also have IP (intellectual property) requirements on the studies performed by their own researchers.\n",
+    "\n",
+    "Some framework disables the BYOC feature and only allowed the pre-installed package during production, but in research and developement period, researchers needs to modify the code constantly during experiments. \n",
+    "\n",
+    "\n",
+    "NVFLARE comes with an authorization system that can help address these security concerns and IP requirements. With this system, an organization can define strict policy to control access to their computing resources and/or FL jobs.\n",
+    "\n",
+    "Here are some examples that an org can do:\n",
+    "\n",
+    "* Restrict BYOC to only the org’s own researchers;\n",
+    "\n",
+    "* Allow jobs only from its own researchers, or from specified other orgs, or even from specified trusted other researchers;\n",
+    "\n",
+    "* Totally disable remote shell commands on its sites\n",
+    "\n",
+    "* Allow the “ls” shell command but disable all other remote shell commands\n",
+    "\n",
+    "\n",
+    "### Centralized vs. Federated Authorization\n",
+    "\n",
+    "Instead of relying a centralized (FL Server) to authorizate and authenticate the user, NVFLARE now uses federated authorization where each organization defines and enforces its own authorization policy:\n",
```

Comment:
"Instead of relying on a centralized FL Server to authorize and authenticate users, NVFLARE uses federated authorization, where each organization defines and enforces its own authorization policy:"